framework_tvb/tvb/interfaces/web/static/js/bursts.js

```diff
@@ -936,6 +936,54 @@ function displayBurstTree(selectedHref, selectedProjectID, baseURL) {
     $("#div-burst-tree").show();
 }
 
+function displayPseSimulationMessage(simulation_mesage){
+    pse_param1_lo = $("#pse_param1_lo")[0].valueAsNumber;
+    pse_param1_hi = $("#pse_param1_hi")[0].valueAsNumber;
+    pse_param1_step = $("#pse_param1_step")[0].valueAsNumber;
+
+    param1_difference = pse_param1_hi - pse_param1_lo;
+    pse_param1_number = Math.floor( param1_difference / pse_param1_step);
+    remainder_param1 = param1_difference % pse_param1_step;
+    if(remainder_param1 !== 0){
+        pse_param1_number = pse_param1_number + 1;
+    }
+
+    pse_param2_lo = $("#pse_param2_lo");
+
+    if(pse_param2_lo.length !== 0) {
```

Comment:
This looks like duplicate (what is done for both param1 and param2. 
Can't you extract a reusable function?